lab/skel_graph/traversal/problems/alee/main.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 Nicolae Mara-Ioana <maraioana9967@gmail.com>
```

Comment:
Please check that you have a duplicate source code. Delete alee.cpp and if the reported issues are still available, fix them in main.cpp